prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -138,17 +139,25 @@ if [[ "$BUILD_TYPE" == "pr" ]]; then
     # In case of PR, operate on PR number
     COMMON_NAME=$(echo "gkeint-pr-${PULL_NUMBER}-${RANDOM_NAME_SUFFIX}" | tr "[:upper:]" "[:lower:]")
     KYMA_INSTALLER_IMAGE="${DOCKER_PUSH_REPOSITORY}${DOCKER_PUSH_DIRECTORY}/gke-integration/${REPO_OWNER}/${REPO_NAME}:PR-${PULL_NUMBER}"
+    export KYMA_INSTALLER_IMAGE
+elif [[ "$BUILD_TYPE" == "release" ]]; then
+    readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+    readonly RELEASE_VERSION=$(cat "${SCRIPT_DIR}/../../RELEASE_VERSION")
+    echo "Reading release version from RELEASE_VERSION file, got: ${RELEASE_VERSION}"
+    readonly COMMIT_ID=$(cd "$KYMA_SOURCES_DIR" && git rev-parse --short HEAD)
+    COMMON_NAME=$(echo "gkeint-commit-${COMMIT_ID}-${RANDOM_NAME_SUFFIX}" | tr "[:upper:]" "[:lower:]")
```

Comment:
Consider a specific prefix for release, to differentiate from master builds: `gkeint-release-${COMMIT_ID}`.
It would make resources created for the purpose of the release more visible. 
The only drawback is that we then have to extend regexp(s) in cleanup jobs to match the new pattern.